datalab/datalab_session/data_operations/median.py

```diff
@@ -42,63 +40,40 @@ def wizard_description():
             }
         }
     
-    def operate(self):
-        input_files = self.input_data.get('input_files', [])
-        file_count = len(input_files)
+    def operate(self, input_files, cache_key):
 
-        if file_count == 0:
-            return { 'output_files': [] }
+        log.info(f'Executing median operation on {len(input_files)} files')
 
-        log.info(f'Executing median operation on {file_count} files')
+        image_data_list = load_image_data_from_fits_urls(input_files)
 
-        with tempfile.TemporaryDirectory() as temp_dir:
-            memmap_paths = []
+        self.set_percent_completion(0.4)
```

Comment:
Where does this 0.4 come from?

Reply:
Im not really sure how to record the progress since I split the program up into utility functions, so I did it by estimating